Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -211,110 +232,113 @@ class PreplannedMapModelTests: XCTestCase {
         }
         
         let mockArea = MockPreplannedMapArea()
-        let model = PreplannedMapModel(offlineMapTask: OfflineMapTask(onlineMap: Map()), mapArea: mockArea, portalItemID: "")!
+        let model = PreplannedMapModel(
+            offlineMapTask: OfflineMapTask(onlineMap: Map()),
+            mapArea: mockArea,
+            portalItemID: "",
+            preplannedMapAreaID: ""
+        )
         await model.load()
         
         guard case .packageFailure = model.status else {
             XCTFail("PreplannedMapModel status is not \".loadFailure\".")
             return
         }
-        
-        XCTAssertFalse(model.canDownload)
-        XCTAssertTrue(model.status.needsToBeLoaded)
     }
     
     @MainActor
-    func testDownloadingStatus() async throws {
-        class MockPreplannedMapArea: PreplannedMapAreaProtocol {
-            func retryLoad() async throws {}
+    func testDownloadStatuses() async throws {
```

Comment:
Without creating a *Many* more protocols for dependency injection, the only other way to test this Model well is to simply test against a live service. This test runs quickly and if the toolkit is using DejaVu, it can be mocked and ran even more quickly.